samples/ErniAcademy.Events.Samples/Program.cs

```diff
@@ -1,15 +1,18 @@
 ﻿using ErniAcademy.Events.Samples;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
 
 IServiceCollection services = new ServiceCollection();
 
 var configuration = new ConfigurationBuilder()
     .SetBasePath(Environment.CurrentDirectory)
-    .AddJsonFile("appsettings.json", optional: false)
+    .AddJsonFile("appsettings.development.json", optional: false)
```

Comment:
will only use the develop json file? i mean seems like you delete the prod.json and add .development.json file!